### ISSUE TITLE
Disable highlighted button status of cancel button at login window install.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gonchik/munki#20]
```
What steps will reproduce the problem?
1. Install updates at login window.
2. Cancel button is highlighted.
3. Accidentally pressing of the space bar will activate the cancel button and 
stop any running updates.

What is the expected output? What do you see instead?
Pressing the cancel button should be an active task for the user, with the 
button highlighted accidentally triggering the button is possible.

What version of the product are you using? On what operating system?
6.0.664. Snow Leopard

Please provide any additional information below.

```

Original issue reported on code.google.com by `christophergrande` on 17 Aug 2010 at 4:13

**Comment by GoogleCodeExporter** [comment]
```
I don't consider this a defect, as it is operating as designed and intended: I 
intended to allow people to be able to cancel a session.  Activating the cancel 
button with the spacebar is standard Apple control behavior, since there is 
only one active control. Perhaps there is a way to disable this behavior -- 
I'll try to find out.

I had put some code in place to allow the site administrator to disable the 
Cancel button (and therefore remove the ability for end-users to cancel an 
install session) by adding a preference key to 
/Library/Preferences/ManagedInstalls.plist, but I can't seem to find it any 
more. Perhaps I never fully implemented it. As I recall, it would disable the 
Cancel button always, not just at the loginwindow.

Would this functionality be useful to complete?

```

Original comment by `gregnea...@mac.com` on 17 Aug 2010 at 4:55
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
The think having the ability to completely disable the cancel button would be a 
good feature to implement. 
```

Original comment by `christophergrande` on 17 Aug 2010 at 9:19

**Comment by GoogleCodeExporter** [comment]
```
Yes, I vote for option to disable Stop button with a key/value in 
ManagedInstalls.plist
I am using different ManagedInstalls.plist-files for almost every client (or 
group) and prefer that some of them should not be able to cancel managed 
installs.

And as I myself like to use that full keyboard access it sometimes creates 
unwanted results but that is the way it works.
```

Original comment by `miq...@gmail.com` on 17 Aug 2010 at 9:27

**Comment by GoogleCodeExporter** [comment]
```
Spacebar activation of the Stop button is fixed in version 3.0.3 of Managed 
Software Update. (That is to say that the spacebar no longer activates the Stop 
button.) This is available in the SVN repo now, and will be rolled into the 
next munki tools package release.
```

Original comment by `gregnea...@mac.com` on 26 Aug 2010 at 5:52
- Changed state: **Fixed**
